taf/src/main/java/com/taf/automation/api/ApiUtils.java

```diff
@@ -310,12 +313,30 @@ public static void aliasField(XStream xStream, Class definedIn, String aliasPref
      * @param aliasPrefix   - the prefix to append to the fields (a colon will be added)
      * @param excludeFields - fields to exclude in the alias configuration
      */
-    public static void aliasField(XStream xStream, Class definedIn, String aliasPrefix, Set<String> excludeFields) {
+    public static void aliasField(XStream xStream, Class<?> definedIn, String aliasPrefix, Set<String> excludeFields) {
         for (Field item : FieldUtils.getAllFields(definedIn)) {
             if (!excludeFields.contains(item.getName())) {
                 xStream.aliasField(aliasPrefix + ":" + item.getName(), definedIn, item.getName());
             }
         }
     }
 
+    /**
+     * Get XStream configured to ignore the namespace when processing<BR>
+     * <B>Note: </B> This should be used this a soap response to be able to map to objects with limited annotations
```

Comment:
Grammar issue.  Probably should be "This should be used **with** a soap response to be able to map to objects with limited annotations"